src/platform/amd/renoir/platform.c

```diff
@@ -61,7 +61,7 @@ static const struct sof_ipc_fw_ready ready
 	.flags = DEBUG_SET_FW_READY_FLAGS,
 };
 
-SHARED_DATA struct timer timer = {
+static SHARED_DATA struct timer timer = {
```

Comment:
btw, can these be const ?

Reply:
no, `struct timer` has many fields, only 3 of them are set in platform.c